### PR TITLE
Add database yml config, resolves #20

### LINKED ISSUE #20: Update the keys in database.yml to reflect secrets.yml

### PR DESCRIPTION
**Pull request by d4be4st**

### CODE REVIEW

File: template.rb [hunk 1]
```diff
@@ -196,12 +218,17 @@
...
 
 FIGARO_FILE = <<-HEREDOC.strip_heredoc
+  database_host: localhost
+  database_username: postgres
+  database_password: ""
   bugsnag_api_key: ADD_IT_HERE
 
   development:
     secret_key_base: #{SecureRandom.hex(64)}
+    database_database: #{app_name}_development
```
Comment (DamirSvrtan) [inline template.rb]: wouldn't it make sense to call this key `database_name`?

Comment (d4be4st) [reply template.rb]: I use `database` because that is used in `database.yml`, not `name`

Comment (vr4b4c) [reply template.rb]: ++ for `database_name`

Comment (DamirSvrtan) [reply template.rb]: Let's change this to databae_name and merge it in.

File: template.rb [hunk 2]
```diff
@@ -177,6 +194,11 @@
 SECRETS_YML_FILE = <<-HEREDOC.strip_heredoc
   default: &default
     secret_key_base: <%= Figaro.env.secret_key_base! %>
+    database:
+      database: <%= Figaro.env.database_name! %>
```
Comment (DamirSvrtan) [inline template.rb]: You have to change it here also @d4be4st

**Pull request merged**